pkg/scheduler/factory/factory.go

```diff
@@ -483,17 +483,21 @@ func (c *configFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 
 // getBinderFunc returns a func which returns an extender that supports bind or a default binder based on the given pod.
 func getBinderFunc(client clientset.Interface, extenders []algorithm.SchedulerExtender) func(pod *v1.Pod) Binder {
-	var extenderBinder algorithm.SchedulerExtender
+	var extenderBinders []algorithm.SchedulerExtender
 	for i := range extenders {
 		if extenders[i].IsBinder() {
```

Comment:
How about adding `IsInterested` logic here?

`if extenders[i].IsBinder()` => `if extenders[i].IsBinder() && extenderBinders[i].IsInterested(pod)`

Reply:
That was my first attempt.
However, Pod parameter is for the func to be returned, not getBinderFunc

Reply:
Get it now!